templates/requirements.txt.j2

```diff
@@ -0,0 +1,5 @@
+docker-py{{ '' if docker_py_version == 'latest' else ('==' + docker_py_version) }}
+dockerrotate=={{ docker_rotate_version }}
+{% if docker_compose_version | bool %}
```

Comment:
In my standalone testing with a toy playbook and ansible 1.9.3 and 2.1.1, `"latest" | bool` is false, skipping docker-compose. Did you find different?

So `if docker_compose_version is not none` here and use `null` above for no docker-compose?